### PR TITLE
Sensision runner

### PR DESCRIPTION
**Pull request by steven-gueguen**

### CODE REVIEW

File: warp10/src/main/sh/warp10.sh [hunk 1]
```diff
@@ -212,6 +212,18 @@ postInit() {
...
 
+
+  echo "
+//
+// Uncomment the following lines to enable metrics collections.
+// Report here the result of the token generation with the envelope ${WARP10_HOME}/tokens/sensision-tokengen.mc2
+//
+// SENSISIONEVENT, SENSISIONGET, SENSISIONSET, SENSISIONUPDATE
+#warpscript.extension.sensision = io.warp10.script.ext.sensision.SensisionWarpScriptExtension
+#sensisionReadToken@/sensision=
```
Comment (hbs) [inline warp10/src/main/sh/warp10.sh:223]: be more specific in the path, only allowing the script we added to access the tokens

File: etc/sensision-tokengen.mc2 [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+//
+
+//
+// This is an envelope to generate sensision tokens.
+//
+
+JSONPRETTY
+
+'sensision' 'applicationName' STORE
+NOW 100 ADDYEARS 'expiryDate' STORE
```
Comment (hbs) [inline etc/sensision-tokengen.mc2:24]: Set the issuance and expiry to fixed values so token generation is deterministic for a given value of IDENT.

Explain that the values are chosen so multiple calls to tokengen sensision-tokengen.mc2 lead to the same token.

**Review by hbs — CHANGES_REQUESTED** [review]

**Comment by steven-gueguen** [comment]
PTAL

**Comment by hbs** [comment]
LGTM

**Review by hbs — APPROVED** [review]

**Pull request merged**